worker/stripe_charge_succeeded.js

```diff
@@ -11,10 +30,14 @@ module.exports = function(notifier_messager, mailroom) {
     }
 
     var currency_code = event.data.currency.toUpperCase();
+    // If this is a zero decimal currency then use it directly
+    // Otherwise divide by 100 to get currency fractional value
+    var amount = zeroDecimalCurrencies.indexOf(currency_code) !== -1 ?
```

Comment:
I always preferred >= 0 in indexOf checks because I find the lack of a double negative makes it easier to understand and read.